aodncore/pipeline/handlerbase.py

```diff
@@ -640,24 +647,29 @@ def _handle_error(self, exception, full_traceback=False):
         self._error = exception
         self._result = HandlerResult.ERROR
 
+        should_notify = []
+        notify_params_dict = self.notify_params or {}
+
         try:
-            if full_traceback:
+            if system_error:
                 self.logger.exception(format_exception(exception))
 
                 import traceback
                 self._error_details = traceback.format_exc()
 
                 # invalid configuration means notification is not possible
-                if isinstance(exception, (InvalidConfigError, MissingConfigParameterError)):
-                    self.notify_on_error = self.notify_on_success = False
-                    self.notify_params = {'error_notify_list': []}
-                else:
-                    self.notify_on_error = True
-                    self.notify_params = {
-                        'error_notify_list': self.config.pipeline_config['global']['admin_recipients']}
+                if not isinstance(exception, (InvalidConfigError, MissingConfigParameterError)):
+                    should_notify.extend(notify_params_dict.get('owner_notify_list', []))
+
             else:
                 self.logger.error(format_exception(exception))
                 self._error_details = str(exception)
+                should_notify.extend(notify_params_dict.get('error_notify_list', []))
+
+                if notify_params_dict.get('notify_owner_error', False):
```

Comment:
I think the owner should *always* be notified of errors, or at least the default here should be `True`, with the option to turn it off in the config if the owner really wants (... to stick his/her head in the sand... )

Reply:
The distinction is between the "system errors" and the "user uploaded the wrong thing and should have known better" processing errors.

System errors always get sent, so this is to have the option to mute some things *if* desired.

I think changing the default to True is my preferred option, to maintain the balance between sensible default but maintaining flexibility.

Reply:
Ah right, that makes sense.
We definitely won't want `notify_owner_success` to be be True by default, so maybe we should just leave both defaults as False.

Reply:
I'm not fussed either way, in practical terms, it will just affect what *typically* needs to be in a watch config.

If *most* pipelines want the errors then maybe it makes sense to have True as default, and save having to configure it explicitly in every watch. Or vice versa!

Reply:
Ok, leave both defaults as False. I think in general we all want to receive less email!